cypress/e2e/rooms/changeRoomsPermission/changeRoomPermissionViewerToAdmin.feature

```diff
@@ -0,0 +1,99 @@
+@regression_test
+@stable_test
+Feature: Room - Change room permission (Viewer - Admin)
+
+    As a teacher, I want to change a participant’s room permission from viewer to admin, so that other users can access the room and leave it as needed
+
+    Scenario: Teacher change room permission of other participants, including pre-conditions
+        Given I am logged in as a '<teacher_2>' at '<namespace>'
+        Given I am logged in as a '<teacher_1>' at '<namespace>'
+
+        # pre-condition: first teacher creating a new room
+        When I go to room overview
+        When I click on FAB to create new room
+        Then I see room creation page
+        When I enter the room name '<room_name>'
+        When I click on the button to save the room
+        Then I see the detail page of room '<room_name>'
```

Comment:
you could use the precondition methods ins PrePostConditionSteps.spec.js to keep the feature file focussed on the feature steps.

Reply:
done